app/models/manageiq/providers/kubevirt/infra_manager/provision/cloning.rb

```diff
@@ -82,4 +79,111 @@ def start_clone(options)
     # be used to check if it has been already created:
     phase_context[:new_vm_ems_ref] = offline_vm.metadata.uid
   end
+
+  def create_persistent_volume_claims(pvcs, params, namespace)
+    pvcs.each do |pvc|
+      pvc = param_substitution!(pvc, params)
+
+      pvc.deep_merge!(
+        :metadata => {
+          :namespace => namespace
+        }
+      )
+
+      # Send the request to create the persistent volume claim:
+      source.with_provider_connection do |connection|
+        connection.create_pvc(pvc)
+      end
+    end
+  end
+
+  private
+
+  def offline_vm_from_objects(objects)
+    vm = nil
+    objects.each do |object|
+      if object.kind == "OfflineVirtualMachine"
+        vm = to_hash(object)
+      end
+    end
+    # make sure there is one
+    raise MiqException::MiqProvisionError if vm.nil?
+    vm
+  end
+
+  def persistent_volume_claims_from_objects(objects)
+    pvcs = []
+    objects.each do |object|
+      if object.kind == "PersistentVolumeClaim"
+        pvcs << to_hash(object)
+      end
+    end
+    pvcs
+  end
+
+  def values(template, options)
```

Comment:
please add doc. I'd suggest to do that for the other methods as weel

Reply:
Sure, Will make sure that docs is provided for new code in this PR.